frontend/packages/dev-console/integration-tests/support/pageObjects/helm-po.ts

```diff
@@ -24,4 +24,7 @@ export const helmPO = {
   uninstallHelmRelease: {
     releaseName: '#form-input-resourceName-field',
   },
+  sidebarPage: {
```

Comment:
It's not a page, maybe we can update it as sideBar or sidePane

Reply:
used sidePane instead of sidebarPage, thank you

Reply:
done, https://github.com/shvasude/console/commit/0e875a464a8fd0a089b59ee6809a182c9c859cab